plugin/emacs/flycheck-quicklintjs.el

```diff
@@ -45,23 +46,34 @@
 (flycheck-def-args-var flycheck-quicklintjs-args javascript-quicklintjs)
 (flycheck-def-executable-var javascript-quicklintjs "quick-lint-js")
 
+(defun flycheck-quicklintjs-parse-errors (output checker buffer)
+  "Parse quick-lint-js alist output format from OUTPUT"
+  (let (err-accum)
+    (dolist (l (car (read-from-string output)))
+      (let ((region (nth 0 l))
+            (sev (nth 1 l))
+            (code (nth 2 l))
+            (msg (nth 3 l)))
+        (push (flycheck-error-new-at-pos
+               (car region)
+               (if (= sev 0) 'error 'warning)
+               msg
+               :id code
+               :buffer buffer
+               :checker checker
+               :end-pos (cdr region)) err-accum)))
+    err-accum))
```

Comment:
Is there a `map` function we can use instead of manually `push`ing onto a temporary list? Something like this:
```suggestion
  (map (car (read-from-string output)) (lambda (l)
      (let ((region (nth 0 l))
            (sev (nth 1 l))
            (code (nth 2 l))
            (msg (nth 3 l)))
        (push (flycheck-error-new-at-pos
               (car region)
               (if (= sev 0) 'error 'warning)
               msg
               :id code
               :buffer buffer
               :checker checker
               :end-pos (cdr region)) err-accum))))
```